apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsTaggedControl.tsx

```diff
@@ -0,0 +1,71 @@
+import { FormControl, VStack } from "@chakra-ui/react"
+import {
+  and,
+  ControlProps,
+  RankedTester,
+  rankWith,
+  schemaMatches,
+} from "@jsonforms/core"
+import { withJsonFormsControlProps } from "@jsonforms/react"
+import { FormLabel, MultiSelect } from "@opengovsg/design-system-react"
+import { ArticlePagePageProps } from "@opengovsg/isomer-components"
+
+import { JSON_FORMS_RANKING } from "~/constants/formBuilder"
+import { collectionItemSchema } from "~/features/editing-experience/schema"
+import { useQueryParse } from "~/hooks/useQueryParse"
+import { trpc } from "~/utils/trpc"
+
+export const jsonFormsTaggedControlTester: RankedTester = rankWith(
+  JSON_FORMS_RANKING.ArrayControl,
```

Comment:
nit: can give it a separate ranking constant, e.g. `JSON_FORMS_RANKING.TaggedControl`